From 3042856cba215a54c5f1c24182c447158ed20748 Mon Sep 17 00:00:00 2001
From: Thomas Braun <thomas.braun@byte-physics.de>
Date: Fri, 27 Nov 2020 16:16:42 +0100
Subject: [PATCH 1/1] WIP: fast sphinx rst update cyle

---
 Packages/doc/conf.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Packages/doc/conf.py b/Packages/doc/conf.py
index 57f5ff8e1..f463b301a 100644
--- a/Packages/doc/conf.py
+++ b/Packages/doc/conf.py
@@ -16,11 +16,11 @@ def get_version():
     return "({branch}) {version}".format(branch=branchString.decode('ascii'), version=revString.decode('ascii'))
 
 # sphinx config
-extensions = ['sphinx.ext.mathjax', 'sphinx.ext.todo', 'breathe', \
-              'sphinxcontrib.fulltoc', 'sphinxcontrib.images',    \
+extensions = ['sphinx.ext.mathjax', 'sphinx.ext.todo',         \
+              'sphinxcontrib.fulltoc', 'sphinxcontrib.images', \
               'sphinxembeddedvideos.youtube']
 
-master_doc = "index"
+master_doc = "user"
 project= "MIES Igor"
 
 html_show_sourcelink = False
-- 
2.29.1.windows.1

